examples/schedule_identical_transaction/main.go

```diff
@@ -0,0 +1,218 @@
+			return
+		}
+
+		// Make sure the transaction succeeded
+		transactionReceipt, err := createResponse.GetReceipt(client)
+		if err != nil {
+			println(err.Error(), ": error getting receipt 1")
+			return
+		}
+
+		newClient, err := hedera.ClientForName(os.Getenv("HEDERA_NETWORK"))
+		if err != nil {
+			println(err.Error(), ": error creating client")
+			return
+		}
+		newClient = newClient.SetOperator(*transactionReceipt.AccountID, newKey)
+
+		clients[i] = newClient
+		accounts[i] = *transactionReceipt.AccountID
+
+		fmt.Printf("account = %v\n", accounts[i])
+	}
+
+	// A threshold key with a threshold of 2 and length of 3 requires
+	// at least 2 of the 3 keys to sign anything modifying the account
+	keyList := hedera.KeyListWithThreshold(2).
+		AddAllPublicKeys(pubKeys)
+
+	// We are using all of these keys, so the scheduled transaction doesn't automatically go through
+	// It works perfectly fine with just one key
+	createResponse, err := hedera.NewAccountCreateTransaction().
+		// The key that must sign each transfer out of the account. If receiverSigRequired is true, then
+		// it must also sign any transfer into the account.
+		SetKey(keyList).
+		SetInitialBalance(hedera.NewHbar(10)).
+		Execute(client)
+	if err != nil {
+		println(err.Error(), ": error executing create account transaction")
+		return
+	}
+
+	// Make sure the transaction succeeded
+	transactionReceipt, err := createResponse.GetReceipt(client)
+	if err != nil {
+		println(err.Error(), ": error getting receipt 2")
+		return
+	}
+	thresholdAccount := *transactionReceipt.AccountID
+
+	fmt.Printf("threshold account = %v\n", thresholdAccount)
+
+	for _, client := range clients {
+		operator := client.GetOperatorAccountID().String()
+
+		// Each client creates an identical transaction, sending 1 hbar to each of the created accounts,
+		// sent from the threshold Account
+		tx := hedera.NewTransferTransaction()
+		for _, account := range accounts {
+			tx.AddHbarTransfer(account, hedera.NewHbar(1))
+		}
+		tx.AddHbarTransfer(thresholdAccount, hedera.NewHbar(3).Negated())
+
+		tx, err := tx.FreezeWith(client)
+		if err != nil {
+			println(err.Error(), ": error while freezing transaction for client ", )
+			return
+		}
+
+
+		signedTransaction, err := tx.SignWithOperator(client)
+		if err != nil {
+			println(err.Error(), ": error while signing with operator client ", operator)
+			return
+		}
+
+		scheduledTx, err := hedera.NewScheduleCreateTransaction().
+			SetScheduledTransaction(signedTransaction)
+		if err != nil {
+			println(err.Error(),": error while setting scheduled transaction with operator client", operator)
+			return
+		}
+
+		scheduledTx = scheduledTx.
+			SetPayerAccountID(thresholdAccount)
+
+		response, err := scheduledTx.Execute(client)
+		if err != nil {
+			println(err.Error(), ": error while executing schedule create transaction with operator", operator)
+			return
+		}
+
+		receipt, err := hedera.NewTransactionReceiptQuery().
```

Comment:
@failfmi You'd be required to query the receipt using `NewTransactionReceiptQuery` directly instead of `GetReceipt()` since the object of `GetReceipt()` is to return an error if the status is not `SUCCESS`

Reply:
To be honest, I do not understand why would have two behaviours for almost the same thing, but since this is right way - LGTM!

Reply:
Is this somewhere documented?

Reply:
Unfortunately it isn't, but we really should.